project-api-tests/src/test/resources/bal-tool/cmd-outputs/tool-list-with-multiple-tool-versions.txt

```diff
@@ -0,0 +1,4 @@
+|TOOL ID               |VERSION         |
+|----------------------|----------------|
+|disttest              |  1.1.0         |
+|disttest              |* 1.0.0         |
```

Comment:
With local repository support, we have introduced a new column called `REPO`. Is this test passing?

Reply:
Fixed with 4655a12b0cf589483ba556c28259e22d5beb4126